postgres-driver/load_balancer.go

```diff
@@ -151,7 +151,7 @@ type dbStickinessOptionsJSON struct {
 	Stickiness bool     `json:"stickiness"`
 }
 
-func (j dbStickinessOptionsJSON) toOutput() any {
+func (j dbStickinessOptionsJSON) toOutput() *repository.StickyOptions {
```

Comment:
Great change here: both on removing the `any` and using the existing type. Thank you!